doc/modules/timer.md

```diff
@@ -12,24 +12,54 @@ All timeout or interval in this module are measured in milliseconds.
 
 Also, timeout and interval should be a non-negative integer(the max of integer is 0x7FFFFFFF). If timeout or interval is negative, then it will be reset to zero, e.g. the task will be put in the task queue immediately.
 
-### setTimeout(fnId: number, timeout: number)
+### setTimeout(fn: function, timeout: number)
 Execute a one-time task after a fixed delay.
 #### Arguments
-* `fnId` *(number)*: the task to be executed.
-* `timeout`*(object)*: the time delay when executing task.
+* `fn` *(function)*: the task to be executed.
+* `timeout`*(number)*: the time delay when executing task.
 
-### setInterval(fnId: number, interval: number)
+### setInterval(fn: function, interval: number)
 Execute a repeated task for at a fixed rate.
 #### Arguments
-* `fnId` *(number)*: the task to be executed.
-* `interval`*(object)*: the time interval when executing two tasks.
+* `fn` *(function)*: the task to be executed.
+* `interval`*(number)*: the time interval when executing two tasks.
 
 ### clearTimeout(fnId: number)
 Stop a specified one-time task. If this method is executed before the corresponding task start, the task would be destroyed. Otherwise, this method has no influence on the task.
 #### Arguments
-* `fnId` *(number)*: the return value of corresponding `setTimeout(fnId: number, timeout: number)`.
+* `fnId` *(number)*: the return value of corresponding `setTimeout(fn: function, timeout: number)`.
 
 ### clearInterval(fnId: number)
 Stop a specified repeated task.
 #### Arguments
-* `fnId` *(number)*: the return value of corresponding `setInterval(fnId: number, interval: number)`.
\ No newline at end of file
+* `fnId` *(number)*: the return value of corresponding `setInterval(fn: function, interval: number)`.
+
+## Example
+	<template>
```

Comment:
Please use triple ` at the begin and end instead of 4-space text intend and specify the highlight language as "html". For example

    <div>hello</div>

and 

```html
<div>hello</div>
```